.github/scripts/bal_proposals_bot/target/build

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@bimsarapathiraja Shall we put `target` into ignore list and remove the files committed in `target`